src/main.ts

```diff
@@ -21,9 +21,14 @@ function ensure<T, K extends keyof T>(target: T, key: K): T[K] {
 	return target[key];
 }
 
-export interface LanguageClientOptions extends VSLanguageClientOptions {
+export interface ISqlopsFeature {
```

Comment:
other places are using SqlOps casing, I think we should be consistent and use ISqlOps... here.